app/javascript/components/redux/greetingStore.js

```diff
@@ -0,0 +1,25 @@
+const apiUrl = "http://localhost:3000/api/greetings";
```

Comment:
- [ ] You do not have a route defined as `api/greetings`. What you have is `api/messages`

Reply:
done